platform/linux-generic/pktio/dpdk.c

```diff
@@ -956,17 +966,24 @@ static inline int pkt_to_mbuf_zero(pktio_entry_t *pktio_entry,
 		if (odp_likely(pkt_hdr->seg_count == 1)) {
 			mbuf_update(mbuf, pkt_hdr, pkt_len);
 
-			if (odp_unlikely(pktio_entry->s.enabled.chksum_insert))
+			if (odp_unlikely(chksum_enabled))
 				pkt_set_ol_tx(pktout_cfg, pktout_capa, pkt_hdr,
 					      mbuf, odp_packet_data(pkt));
 		} else {
+			int dummy_idx = 0;
+
 			/* Fall back to packet copy */
 			if (odp_unlikely(pkt_to_mbuf(pktio_entry, &mbuf,
-						     &pkt, 1) != 1))
+						     &pkt, 1, &dummy_idx) != 1))
 				goto fail;
 			(*copy_count)++;
 		}
 		mbuf_table[i] = mbuf;
+
+		if (odp_unlikely(tx_ts_enabled)) {
```

Comment:
I am not sure this should be marked unlikely.

Reply:
Removing the `odp_unlikely()` causes a measurable performance penalty in the common processing path.